### ISSUE TITLE
🐛 Bug: "Screencast from ..." not detected as default alt text

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [github/accessibility-alt-text-bot#47]
Coming over from https://github.com/JoshuaKGoldberg/eslint-plugin-expect-type/issues/155#issue-2151584953, someone posted a comment like:

> ```md
> ![Screencast from 23 02 2024 19 15 19](https://github.com/JoshuaKGoldberg/eslint-plugin-expect-type/assets/540079/8c8da5f5-0656-48d6-98e1-7fe179bb7ef7)
> ```

I think that alt text should be considered invalid, right?

**Comment by mohammee1** [comment]
QMGVXZNYCCRVFTAWKTB242KTM6ZLXQMN

**Comment by mohammee1** [comment]
QMGVXZNYCCRVFTAWKTB242KTM6ZLXQMN
